lib/api/links_builder.rb

```diff
@@ -42,7 +42,10 @@ def format_href(new_offset)
       if href.include?("offset")
         href.sub("offset=#{offset}", "offset=#{new_offset}")
       else
-        href + "&offset=#{new_offset}"
+        result = URI.parse(href)
+        query_params = result.query.to_s.split("&") + ["offset=#{new_offset}"]
+        result.query = query_params.join("&")
+        result.to_s
```

Comment:
you'd think URI would have a method to append something to a query.

maybe something like this:

```
result = URI.parse(href)
result.query = [result.query, "offset=#{new_offset}"].compact.join('&')
result.to_s
```

Thanks.